src/pages/my_page/user_profile/UserOrcid.tsx

```diff
@@ -31,10 +32,14 @@ export const UserOrcid = ({ user }: UserOrcidProps) => {
   const [isAddingOrcid, setIsAddingOrcid] = useState(false);
   const [isRemovingOrcid, setIsRemovingOrcid] = useState(false);
   const userCristinId = user.cristinId ?? '';
-  const [cristinPerson, isLoadingCristinPerson, refetchCristinPerson] = useFetch<CristinPerson>({
-    url: userCristinId,
-    errorMessage: t('feedback.error.get_person'),
+
+  const cristinPersonQuery = useQuery({
+    queryKey: [userCristinId],
```

Comment:
Ser at `userCristinId` kan være tom streng fra linje 34. Hva vil skje her da? Usikker på når/hvorfor det kan skje, da man vel ikke skal kunne komme inn om man ikke har cristin-bruker 🤔

Reply:
Samme gjelder for `ResearchProfile` ser jeg, så sikkert greit nok. Kunne evt hatt en `enabled: userCristinId` eller noe, for å unngå å sende et kall når/om det mangler. Men ser litt ut som at det er et tenkt scenario